.readthedocs.yml

```diff
@@ -8,6 +8,9 @@ python:
     - requirements: requirements/docs.txt
 
 search:
+  ignore:
+    # Internal documentation
+    - development/design/*
```

Comment:
the defaults `search.html` and `404.html` should be added here (probably the example in the docs should include this to make it easy to copy/paste).

Also, you can remove this from search.ranking now.